com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -229,6 +244,24 @@ public void ConfigureInput(ScriptableRenderPassInput passInput)
             m_Input = passInput;
         }
 
+        public void ConfigureColorStoreAction(uint attachmentIndex, RenderBufferStoreAction storeAction)
+        {
+            m_ColorStoreActions[attachmentIndex] = storeAction;
+        }
+
+        public void ConfigureColorStoreActions(RenderBufferStoreAction[] storeActions, uint attachmentsCount)
```

Comment:
Wouldn't it make more sense to just pass a shorter array in this call? as in, remove attachmentsCount var and just check storeActions.Length?